llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.cpp

```diff
@@ -1724,8 +1724,14 @@ SDValue WebAssemblyTargetLowering::LowerGlobalAddress(SDValue Op,
     fail(DL, DAG, "Invalid address space for WebAssembly target");
 
   unsigned OperandFlags = 0;
-  if (isPositionIndependent()) {
-    const GlobalValue *GV = GA->getGlobal();
+  const GlobalValue *GV = GA->getGlobal();
+  // is PIC but not a WebAssembly table.
+  // Since WebAssembly tables cannot yet be shared accross modules, we don't need special
+  // treatment for tables in PIC mode.
```

Comment:
Are you saying that shouldAssumeDSOLocal() is always true for table symbols?


Why can't tables be imported/exported like any other symbol?

Reply:
As it stands tables need to be always declared with static. Symbols cannot be exported. I will get around to implementing it at some point to fix https://github.com/llvm/llvm-project/issues/64532.

Reply:
I will wait for a further thumbs up from you if you're happy with introducing this sed pattern into the tests.